docs/blog/newsletter-march-2019.rst

```diff
@@ -0,0 +1,111 @@
+- Base OKRs on your goals for the docs. For example, reducing the number of questions that Customer Support handles.
+- If you aren't as concerned about measurability, think about OKRs in terms of answering the questions "Where are we going?" (the objective) and "Are we there yet?" (the key result).
+
+----------------------------------------
+The whys and wherefores of cover letters
+----------------------------------------
+
+A question about how detailed a cover letter should be, posted in the ever-popular `#career-advice <https://writethedocs.slack.com/messages/C6ADX1YVA/`_ channel, produced a lively discussion. While it started with what you should put in a cover letter, it expanded to the purposes of a cover letter, when or whether to include one, and more.
+
+Some folks expressed aversion to cover letters, especially if they're required as part of a longer, more detailed application process: they're a lot of work, and it's not always clear that anyone reads them. But others - hiring managers in particular - really appreciated them.
+
+In general, people seemed to agree that it's a good idea to write at least a brief cover letter that focuses on the job posting and shows how your skills, background, and interests are relevant. But as always - it depends! Do you have a connection or a referral, or some other personal or specific relationship to the job you're applying for? Then a "warm intro" that shows you know something about the job and the company is pretty straightforward. On the other hand, a "cold intro" where you have no particular knowledge or insight might make a cover letter seem all the more useful - but also much harder to tailor to the specific job.
+
+It was also pointed out that a cover letter can be a great way to explain your sideways, unexpected, or less-than-conventional entry into the position you're applying for - often the case for docs-related jobs.
+
+----------------------------------------------
+Moving to docs-as-code: static site generators
+----------------------------------------------
+
+It's no secret that many of us are keen on the docs-as-code approach. But as someone asked this month: if you're moving from a traditional authoring tool like Flare, and want to start storing docs in Git, what should you use to create the docs themselves?
+
+Static site generators are a sensible answer. But how to choose one? `There are so many <https://www.staticgen.com/>`_! The community had a few suggestions to help out: one was to check out `O'Reilly's free intro to SSGs <https://learning.oreilly.com/library/view/static-site-generators/9781492048558/>`_. There were also few specific recommendations:
+
+- `Antora <https://antora.org/>`_ (uses AsciiDoc)
+- `Gatsby <https://www.gatsbyjs.org/>`_
+- `Hugo <https://gohugo.io/>`_ (and there are `introductions to Jekyll, Hugo and Sphinx here <https://www.docslikecode.com/learn/>`_)
```

Comment:
This reads odd to me. Not really connected to Hugo. 

What about adding Hugo, Jekyll and Sphinx as their own items in the list, and then mention the intros at the end?

Reply:
Yeah, good shout, I'll fix that